src/components/Process/Detail.tsx

```diff
@@ -0,0 +1,202 @@
+import {
+  Card,
+  CardBody,
+  CardHeader,
+  CardProps,
+  Flex,
+  Grid,
+  GridItem,
+  Tab,
+  TabList,
+  TabPanel,
+  TabPanels,
+  Tabs,
+  Tag,
+  Text,
+  useBreakpointValue,
+  VStack,
+} from '@chakra-ui/react'
+import {
+  ElectionDescription,
+  ElectionHeader,
+  ElectionResults,
+  ElectionSchedule,
+  ElectionTitle,
+} from '@vocdoni/chakra-components'
+import { useElection } from '@vocdoni/react-providers'
+import { ElectionStatus, IElectionInfoResponse, PublishedElection } from '@vocdoni/sdk'
+import { FallbackHeaderImg } from '~constants'
+import { HeroHeaderLayout } from '~src/layout/HeroHeaderLayout'
+import { CopyButton, ReducedTextAndCopy } from '~components/CopyButton'
+import { Trans, useTranslation } from 'react-i18next'
+import { ElectionStatusBadge } from '~components/Organizations/StatusBadge'
+import { PropsWithChildren } from 'react'
+import { OrganizationCard } from '~components/Organizations/Card'
+import { RawContentBox } from '~src/layout/ShowRawButton'
+import { useElectionKeys } from '~queries/processes'
+import { ucfirst } from '~utils/strings'
+
+const Detail = () => {
+  const { election } = useElection()
+  const { t } = useTranslation()
+
+  if (!election || !(election instanceof PublishedElection)) return null
+  const isSmallScreen = useBreakpointValue({ base: true, md: false })
```

Comment:
Hooks cannot be used after a conditional rendering.

Reply:
Cannot understand why my IDE is not complaining about it.. :S

Reply:
That's weird, but check if the IDE does not complain... I'm pretty sure the browser console should complain.